src/net/sourceforge/kolmafia/request/DrinkItemRequest.java

```diff
@@ -737,6 +737,8 @@ public static final void parseConsumption(
 
           if (!responseText.contains("discard the no-longer-frosty")) {
             success = false;
+          } else {
```

Comment:
Note that this (and the below handling for salad fork) doesn't handle the case where someone uses a frosty mug outside of Mafia. That's probably sufficient for now.